lvgl_tft/ili9341.c

```diff
@@ -9,6 +9,7 @@
 #include "ili9341.h"
 #include "disp_spi.h"
 #include "driver/gpio.h"
+#include "esp32/rom/gpio.h"
```

Comment:
Same here

Reply:
Updated all the files with references to the undefined variable. Compiled without error and works.